main.md

```diff
@@ -572,7 +572,76 @@ Cache-Control: no-cache, no-store
 }
 ```
 
-### Token Content {#token_content}
+### Enriched authorization details in Token Response
+
+There are use cases where the authorization server enriches the data in an authorization details object. For example, a client may ask for access to 
+account information but leave the decision about the accounts it will be able to access to the user. The user would select the sub set of accounts she 
+wants the client to entitle to access in the course of the authorization process. In order to allow the client to determine the accounts it is 
+entitled to access, the authorization server will add this information to the respective authorization details object. 
+
+As an example, the requested authorization detail parameter could look like this:
+
+```JSON
+"authorization_details": [
+   {
+      "type": "account_information",
+      "access": {
+         "accounts": [],
+         "balances": [],
+         "transactions": []
+      },
+      "recurringIndicator":true
+   }
+]
+```
+
+The authorization server then would expand the authorization details object and add the respective account identifiers.
+
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-cache, no-store
+
+{
+   "access_token":"2YotnFZFEjr1zCsicMWpAA",
+   "token_type":"example",
+   "expires_in":3600,
+   "refresh_token":"tGzv3JokF0XG5Qx2TlKWIA",
+   "authorization_details":[
+      {
+         "type":"account_information",
+         "access":{
+            "accounts":[
+               {
```

Comment:
This breaks the recommended schema for the "access" field (an array of strings). While that's allowable, it's probably a bad example and precedent. Maybe make all of "accounts", "balances", and "transactions" top-level objects in the request instead?

Reply:
Is there really a recommendation for "access"?https://www.ietf.org/id/draft-ietf-oauth-rar-02.html#section-2.1 has "actions" but not  "access".  https://www.ietf.org/id/draft-ietf-oauth-rar-02.html#section-2.2 does have an "access" with an array of strings but it's under "permissions" and only part of an example meant to illustrate a collision resistant type.

Reply:
Oh you're right, my bad -- I was indeed thinking of `actions` here, so this example can probably stay.